ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -419,7 +428,8 @@ static int HandleVnodeOperation(
             }
             else if (FileFlagsBitIsSet(currentVnodeFileFlags, FileFlags_IsEmpty))
             {
-                if (!TryGetVirtualizationRoot(&perfTracer, context, currentVnode, pid, &root, &vnodeFsidInode, &kauthResult, kauthError))
+                // Prevent system services from enumerating directories as this tends to cause deadlocks with the kauth listeners for Antivirus software
```

Comment:
Minor comment suggestion:  

Here and in the comment above consider switching from the term "enumerating" to "expanding" (or even "expanding directories as part of enumeration").

Although the name of the callback includes "enumerate" it's the expansion that happens in the callback that causes deadlocks.

Reply:
Done.